source/unified-test-format/tests/valid-pass/observeSensitiveCommands.yml

```diff
@@ -38,6 +38,8 @@ createEntities:
 
 tests:
   - description: "getnonce is observed with observeSensitiveCommands=true"
+    runOnRequirements:
```

Comment:
I wasn't sure if it made sense to try to write new tests for `observeSensitiveCommands` that don't use a command which is limited in server version. We also have tests below that cover 4.9+ (hello with speculative authenticate) so we at least cover all server versions with some command. So for now I just skipped these on 6.2+ but let me know if you think we should switch these to some other command.

Reply:
> so we at least cover all server versions with some command.

That seems fine. I don't think we need to replace this for 6.2+.